### DISCUSSION TITLE
Find easily names of tunnels and in/outputs in a complex circuit (Ctrl-F function ?)

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#999] [Q&A] [answered]
My question is perhaps not clear, thus I reformulate it slightly: I have developed several large circuits. Is there a possibility to do a search in a complex circuit with many in- and outputs for various names of tunnels or in/outputs ? (e.g. using ctrl-f, as in a text editor ?) This would be very useful to my opinion.

**Comment by hneemann** [comment]
see #1010

**Comment by hneemann** [comment, marked as answer]
@jongena 
See the latest [pre release](https://infdigital.dhbw-mosbach.de/).

**Reply by jongena** [reply]
Many thanks !! Look forward to the new relaese !
